apify-docs-theme/static/js/custom.js

```diff
@@ -78,8 +78,7 @@ function scrollOpenApiSidebarItemIntoView() {
 function redirectOpenApiDocs() {
     const { hash, pathname, origin } = new URL(window.location.href);
 
-    // TODO change to '/api/v2'
-    if (pathname.replace(/\/$/, '') !== '/api/v2-new') {
+    if (pathname.startsWith('/api/v2') && pathname.startsWith('/api/v2-')) {
```

Comment:
isn't the `pathname.startsWith('/api/v2')` part redundant?

Reply:
right, should be `if (!pathname.startsWith('/api/v2') || pathname.startsWith('/api/v2-')) {`